product_template_websale_search_extended/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3.0 (https://www.gnu.org/licenses/lgpl-3.0.html)
+{
+    "name": "Product Template Websale Search Extended",
```

Comment:
Typo & renaming suggestion:
```suggestion
    "name": "Website Sale Product Search Extended",
```

and addon could be renamed to `website_sale_product_search_extended`, as starts from general (`website_sale`) to particular (product search), what do you think?